public/app/features/dashboard/components/TransformationsEditor/TransformationOperationRow.tsx

```diff
@@ -122,7 +122,6 @@ export const TransformationOperationRow = ({
         )}
         <QueryOperationToggleAction
           title="Debug"
-          disabled={!isOpen}
```

Comment:
I think with the removal of this we can also remove the arg in `renderActions` since `isOpen` is now unused.